src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -803,7 +803,6 @@ std::set<std::vector<element::Type>> jit_hswish_emitter::get_supported_precision
 }
 
 /// IS_INF ///
-
```

Comment:
here and below: please, dont update not related part of the source code

Reply:
I'm not quite sure why, but for some reason this and several other files are being changed in a similar way when the clang format target is enabled in cmake